packages/gatsby-theme-vtex/src/sdk/graphql/request.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  request as baseRequest,
+  RequestOptions as GraphQLRequestOptions,
+} from '@vtex/gatsby-plugin-graphql'
+
+export type RequestOptions = GraphQLRequestOptions
+
+export const request = async <
+  Query extends any = any,
+  Variables extends any = any
```

Comment:
why don't you use any and any?

Reply:
This was copied from [here](https://github.com/vtex/faststore/blob/ea05570a1a7d5472b4eb557e3be2851569415168/packages/gatsby-plugin-graphql/src/index.ts#L20).